frontend/src/components/mine/Incidents/MineIncidentTable.js

```diff
@@ -59,6 +61,56 @@ const columns = (props) => [
       text: action.mine_incident_followup_investigation_type_code,
     })),
   },
+  {
+    title: "Initial Report Documents",
+    dataIndex: "initialDocuments",
+    width: 200,
+    render: (text, record) => (
+      <div title="Initial Report Documents">
+        {record.docs.length === 0 ? (
+          <span>--</span>
+        ) : (
+          record.docs
+            .filter((file) => file.mine_incident_document_type_code === "INI")
```

Comment:
Could we create a helper function for this, since its the same across both columns?

Reply:
I'm not sure it's useful to create a helper function for a single filter. It's one line and the intent is self describing. That said, maybe defining a constant for our INI/FIN codes would be a good idea?

Reply:
This isn't one line, its around 15 and its repeated half the file down but checking against a different document code